rotkehlchen/accounting/structures/base.py

```diff
@@ -30,6 +30,16 @@ class ActionType(DBEnumMixIn):
     ETHEREUM_TRANSACTION = 3
     LEDGER_ACTION = 4
 
+    def __str__(self) -> str:
+        return self.name.lower() + 's'
```

Comment:
Why put plural here?  The request for plural was in the serialization, but I don't agree with it. This describes the type of action and is singular.